src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php

```diff
@@ -319,4 +315,39 @@ protected function isContentTranslationEnabled($bundle) {
     return $this->moduleHandler->moduleExists('content_translation') && \Drupal::service('content_translation.manager')->isEnabled($this->getEntityTypeId(), $bundle);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function applies($object) {
+    $definition = $this->entityManager->getDefinition($this->getDerivativeId());
```

Comment:
we could also do an instanceof EntityInterface && $object->getEntityTypeId() == $this->getDerivativeId(), a bit simpler?